functional-samples/ai.gemini-on-device/manifest.json

```diff
@@ -6,7 +6,10 @@
   "background": {
     "service_worker": "background.js"
   },
-  "permissions": ["sidePanel"],
+  "permissions": ["sidePanel", "aiLanguageModelOriginTrial"],
+  "trial_tokens": ["A6hC1yvazc5QTqSESgEo3uwIUs1KBM4VB93yiTdtgFlFlYNJjf18zmzScpaCepaYpNLaMJ4dNkDps+vbrk7oEw8AAAB9eyJvcmlnaW4iOiAiY2hyb21lLWV4dGVuc2lvbjovL2lkZm1qYmJqY21qamtoYWlvZmltbWNtY29qYmlnbmpmIiwgImZlYXR1cmUiOiAiQUlQcm9tcHRBUElGb3JFeHRlbnNpb24iLCAiZXhwaXJ5IjogMTk4OTQwNzUwMH0="],
```

Comment:
Thanks! Could we also update the README for this sample to explain how to run Chrome with the right flag for this? We should also mention this is a testing token and explain that developers will be able to register in the future.

Reply:
Dealt with in https://github.com/GoogleChrome/chrome-extensions-samples/pull/1337/commits/68b4923306e46ebbbb71cdeb8974db6018c433f7.